src/ledger/LedgerTxnOfferSQL.cpp

```diff
@@ -671,6 +671,8 @@ LedgerTxnRoot::Impl::dropOffers()
            "price            DOUBLE PRECISION NOT NULL,"
            "flags            INT              NOT NULL,"
            "lastmodified     INT              NOT NULL,"
+           "extension        TEXT,"
```

Comment:
should we fix this table? as this is the one table we're going to rebuild for sure, we can do anything we want wrt schema and indexes
at least make this `NOT NULL` as we always set it to something.

Reply:
Fixed.